src/math/warpExposure.cc

```diff
@@ -350,13 +352,326 @@ namespace {
         return std::abs(dSrcA.getX()*dSrcB.getY() - dSrcA.getY()*dSrcB.getX());
     }
 
+
+    template<typename DestImageT, typename SrcImageT>
+    int doWarpCovariance(
+      DestImageT &destImage,                      ///< remapped %image
+      SrcImageT const &srcImage,                  ///< source %image
+      afwMath::detail::PositionFunctor const &computeSrcPos,   ///< Functor to compute source position
+          ///< called with dest row, column; returns source position (as a Point2D)
+      afwMath::WarpingControl const &control,     ///< warping parameters
+      afwImage::Image<double> &covImage,           ///< covariance matrix
+      typename DestImageT::SinglePixel padValue   ///< value to use for undefined pixels
```

Comment:
It may not matter for this throw-away branch, but you might as well set you editor and your habits to 4-space indents rather than 2.